tests/testapps/CTCatalog/CTCatalog/Tests/CTCAffineTransformationTestViewController.mm

```diff
@@ -49,6 +51,12 @@ - (void)drawRect:(CGRect)rect {
     CGContextScaleCTM(context, _scaleCTM.x, _scaleCTM.y);
     CGContextTranslateCTM(context, _translationCTM.x, _translationCTM.y);
 
+    CGAffineTransform tm = CGContextGetTextMatrix(context);
+    tm = CGAffineTransformRotate(tm, _rotationText);
+    tm = CGAffineTransformScale(tm, _scaleText.x, _scaleText.y);
+    tm = CGAffineTransformTranslate(tm, _translationText.x, _translationText.y);
```

Comment:
This translation is being ignored because `SetTextMatrix` does not clobber the tx/ty.